subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -39,7 +39,6 @@ class ApplyPluginIntegSpec extends AbstractIntegrationSpec {
 
     @Issue("GRADLE-2358")
     @FailsWith(UnexpectedBuildFailure)
-    // Test is currently failing
```

Comment:
Fair enough. The annotation should be self-explanatory. Optimally, we'd move that into the `reason` attribute of the `FailsWith` annotation. The reason should actually explain the _real_ reason.